create_release.sh

```diff
@@ -1,14 +1,14 @@
 
 # if a tag hasn't been passed in as a parameter, create a semantic one
-if [ ! -n "$TAG" ]; then
+if [ ! -n "$tag" ]; then
     docker run --rm  -v"$(pwd)":/app  -ti semantic-release  -- semantic-release version
     export pwd=$(pwd)
     export TAG=$(docker run --tty=false --rm  -v$pwd:/app  -i semantic-release -- semantic-release version --noop | grep "Current version: " | cut -d ' ' -f 3 | sed -e "s/\r//")
 else
-    echo "You passed in '$TAG' as the custom tag"
+    echo "You passed in '$tag' as the custom tag"
     # TR == test release
-    echo "We're going to refactor the tag into 'TR-$TAG' to avoid any conflicts."
-    TAG="TR-$TAG"
```

Comment:
we should probably also update references to the $TAG parameter in the Makefile

Reply:
Didn't find any references to this in the Makefile, the `TAG` argument is passed to this script implicitly

Reply:
Didn't find any references to this in the Makefile, the `TAG` argument is passed to this script implicitly

Reply:
I guess i was referring to this line in the doc of the makefile: https://github.com/masschallenge/impact-api/blob/development/Makefile#L147

Reply:
ah, helptext. Missed that, thanks

Reply:
made that change